modules/ROOT/pages/enterprise/server/troubleshoot.adoc

```diff
@@ -108,7 +113,7 @@ Edit the start.ini file to read as follows:
 
 Restart the service and confirm the settings have been applied like so:
 
-image::{baseurl}/images/spell-checking-jetty-settings.png[Spell Checking Server Settings on Jetty]
+image::spell-checking-jetty-settings.png[Spell Checking Server Settings on Jetty]
 
 == Troubleshooting tools: curl
 
```

Comment:
Below the 2 cURL urls shouldn't be escaped.

Reply:
done